### ISSUE TITLE
The error thrown by run() is wrong when the port is blocked

### ISSUE THREAD

**Issue by settlersxp** [Azure/azure-event-hubs-python#102]
https://github.com/Azure/azure-event-hubs-python/blob/737c5f966557ada2cf10fa0d8f3c19671ae96348/examples/send.py#L36

The error shown is "Please confirm target hostname exists" which is incorrect after a very long wait time. Probably the wait time is equal to the timeout I have set, did not bother to check.

Steps to reproduce:
1) Set valid azure credentials and project information
2) Do not set a proxy
3) Block with a firewall the ports 5671 and 5672
4) Try to open a new consumer and notice that the error message is "Please confirm target hostname exists"  instead of anything related to blocked ports.

**Comment by annatisch** [comment]
Thanks for this report @settlersxp!
I would say it's likely that the underlying error raised if the same in this case (something like "endpoint could not be reached") - which is why it isn't distinguished.
I'll see if there's anything that can be done at the uAMQP level to address this :)

**Comment by settlersxp** [comment]
Makes sense, if it's difficult to distinguish the two errors adding the bit with "or make sure the ports 5671 and 5672 are open in the firewall" should do the trick. Afterall some techy people will use the SDK, there is some tolerance to debug and workarounds.

**Comment by yunhaoling** [comment]
This issue has been moved to https://github.com/Azure/azure-sdk-for-python/issues/5309. Please keep tracking the issue there.

---

This project has been moved to the [azure-sdk-for-python](https://github.com/Azure/azure-sdk-for-python) repository, under:

[azure-sdk-for-python/sdk/eventhub/azure-eventhubs](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/eventhub/azure-eventhubs).

This repository is no longer maintained.

For new issues and pull requests, please go to the [azure-sdk-for-python](https://github.com/Azure/azure-sdk-for-python) repository.
